management/guardian.go

```diff
@@ -11,6 +11,16 @@ type MultiFactor struct {
 	TrialExpired *bool `json:"trial_expired,omitempty"`
 }
 
+type MultiFactorPolicies []string
```

Comment:
I'm not sure if we should alias `[]string`. Do you see some advantage to this, or did you take this as an example from a different method in this package?

Reply:
Good point. I just felt it looked more neat since almost all other methods use a struct type, so the method pattern looks similar for most of the api calls in the file. But we can go without it if you think it's better.